includes/core-changes/aspnetcore/3.0/all-shared-framework-removed.md

```diff
@@ -0,0 +1,39 @@
+### Removed Microsoft.AspNetCore.All shared framework
+
+Earlier this year, we announced the `Microsoft.AspNetCore.App` metapackage (see https://github.com/aspnet/Announcements/issues/287). That announcement said:
```

Comment:
This can be condensed considerably. "Earlier this year", "we announced", and quoting the announcement are really all noise. This should be much more succinct.